scripts/package-providers.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -e -x -o pipefail
+
+BIN_DIR=${BIN_DIR:-./bin}
+
+cd /obot-tools
+
+if [ ! -e workspace-provider ]; then
+    git clone --depth=1 https://github.com/gptscript-ai/workspace-provider
```

Comment:
nit: would be nice to be able to change these with an env var or something.

Reply:
yeah ive got that marked down as a future enhancement